migrations/20151117173053_add_datetime_columns.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
```

Comment:
Making the timestamps wider than 1 number is a good idea, but instead of 20151117173053, let's just use 0001, 0002, etc.